src/macroexpand.scm

```diff
@@ -466,11 +466,12 @@
 
 ;; macro expander entry point
 
-(define (julia-expand-macros e)
-  (cond ((not (pair? e))     e)
+(define (julia-expand-macros-limited e max-depth)
```

Comment:
But I will try a default value `(define (julia-expand-macros e (max-depth -1))`. Did not expect this feature.